KsApi/models/graphql/adapters/Project+FetchProjectQueryDataTests.swift

```diff
@@ -119,55 +131,57 @@ final class Project_FetchProjectQueryDataTests: XCTestCase {
       project.creator.avatar.small,
       "https://ksr-qa-ugc.imgix.net/assets/033/846/528/69cae8b2ccc2403e233b5715cb1f869f_original.png?ixlib=rb-4.0.2&blur=false&w=1024&h=1024&fit=crop&v=1623351187&auto=format&frame=1&q=92&s=d0d5f5993e64056e5ddf7e42b56e50cd"
     )
+    XCTAssertEqual(project.creator.erroredBackingsCount, 1)
     XCTAssertEqual(project.creator.id, decompose(id: "VXNlci0xNTMyMzU3OTk3"))
-    XCTAssertNil(project.creator.isEmailVerified)
+    XCTAssertTrue(project.creator.isEmailVerified!)
```

Comment:
Just a heads up: haven't seen many, if any, implicit optionals being used in the project.

Reply:
Yea, using them quite liberally in this test class (over 40 times), because all the data is mocked and never changes (at least in theory). If it does change, the test will fail/crash, but because its' a test I'm okay with force unwrapping. If we do `if let`/`guard` that's a lot of additional conditional logic for something that doesn't have any impact on application logic. I would rarely if ever use a force unwrap in application logic.